src/Webhook/WebhookEventHandlerChain.php

```diff
@@ -74,7 +74,7 @@ public function supports(Event $event): bool
     }
 
     /**
-     * @return list<WebhookHandlerInterface>
+     * @return array<WebhookHandlerInterface>
```

Comment:
```suggestion
     * @return list<WebhookHandlerInterface>
```
Lets keep this and add it to the baseline instead.